packages/babel-plugin-transform-parameters/test/fixtures/parameters/var-same-as-param-closure/input.js

```diff
@@ -0,0 +1,8 @@
+var x = 1
```

Comment:
nit: you can change this test to `exec.js` and use `expect` assertions:

```js
function foo(x, y = function () { x = 2 }) {
  var x = 3
  y()
  expect(x).toBe(3);
}
foo()
expect(x).toBe(1);
```

Babel fixture test runner will apply transform to `exec.js` and evaluate the assertions above.

Reply:
Thanks!